--- cancd-0.1.0/Makefile~	2005-08-18 21:47:48.000000000 +0300
+++ cancd-0.1.0/Makefile	2006-01-06 21:51:29.000000000 +0200
@@ -1,7 +1,8 @@
 
 VERSION=0.1.0
 
-CFLAGS += -Wall -g -O2 -DVERSION="\"$(VERSION)\""
+OPTFLAGS = -g -O2
+CFLAGS += -Wall $(OPTFLAGS) -DVERSION="\"$(VERSION)\""
 
 cancd: cancd.o
 
@@ -10,8 +11,8 @@
 	cp cancd $(DESTDIR)/usr/sbin
 	chmod 0755 $(DESTDIR)/usr/sbin/cancd
 	mkdir -p $(DESTDIR)/etc/init.d
-	cp cancd.init $(DESTDIR)/etc/init.d/cancd
-	chmod 0755 $(DESTDIR)/etc/init.d/cancd
+	cp cancd.init $(DESTDIR)/etc/rc.d/init.d/cancd
+	chmod 0755 $(DESTDIR)/etc/rc.d/init.d/cancd
 
 clean:
 	rm cancd cancd.o
